<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width-device-widge, initial">
    <title>Gene-ius Project</title>
    <link rel="stylesheet" href= "nav.css" >
    <link rel="stylesheet" href= "Results.css">
     
    
</head>
<style>
    header {
        background-color: #b5a1ca;
        padding-top: 50px;
        padding-bottom: 50px;
        margin: -10px -10px 0px -10px;
    }

    .pfp {
        margin-left: 500px;
        height: 240px;
        width: 30.3%;
    }

    .heading {
        text-align: center;
        font-size: 30px;
        font-family: Arial, Helvetica, sans-serif;
        font-weight: bold;
    }
    .description {
        text-align: center;
        font-size: 21px;
        font-family: Roboto, sans-serif;
    }
</style>
<body>
    <header>

        <img class= "pfp" src="logo4.png">
    </header>
        <div class="navbar">
            <a class="nav-link nav-link-ltr" href="Background.html">Background</a>
            <a class="nav-link nav-link-ltr" href="Goals.html">Goals</a>
            <a class="nav-link nav-link-ltr" href="Code.html">Code</a>
            <a class="active-link" href="Results.html">Results</a>
            <a class="nav-link nav-link-ltr" href="nextsteps.html">Next Steps</a>
            <a class="nav-link nav-link-ltr" href="aboutus.html">About Us</a>
        </div>
    <main>
        <h2> Results </h2>
       
            <p>Overall in research what we found was the largest repeats units of spider spidroins, as well as that when aligning the different repeats that it is more indicative to look and align the spidroin types over the species types because there is more relatedness between the spidroin type. With our results one is also able to see how likely the evolution of the spidroins happened going from each species as well as spidroin type. In addition through our code we were able to make fasta files for the six protein sequences, the largest repeat unit found through our code.</p>
            <div class="dataimg"> 
            <h3>Total Data</h3>
                <div class= "box1">
                    <img src="/total_tree.png" alt="imagept2" >
                    <img src="/total_alignment.png"  alt="image_totalalignment" ></div>
                <p>Through these images one can see the connection between all of the species and spidrons types that were looked at. One is able to predict the way in which evolution happened through the phylogenetic tree (Most likely started with Aarg_MaSp2.1a and the newest evolution is most likely Atri_MaSp2.1b). In the alignment one is also able two see the relativity between the spidrons types and the spider species on a grand scale. One is also able to see in which repeats sequences share more common ground and which are likely to be the farthest away from each other in terms of relativity. For example, Atri_MaSp2.1a and Aaur_MaSp2.1a are the most related even though they are different species but same spidroin type however Atri_MaSp2.1a and  Atri_MaSp2.1b even though same species different spidroin type are less related.
                    </p>
                
            <h3>Spidroin Comparison</h3>
                    <div class= "box2">
                        <img src="/masp1a_aligncomp.png" alt="atree"></div>
                    <div class = "box2-1">
                        <img src="/masp1a_treecomp.png" alt="acompare">
                        <img src="/masp1b_aligncomp.png" alt="btree"></div>
                        <img src="/masp1b_treecomp.png" alt="bcompare">
                    </div>
                    <p> These images are the results of when we compared the spidroins  types' repeat units with each other (1a & 1b). In this part of the data there is a  phylogenetic tree which represent again the likelihood how evolution between different species with the same spidroin type occurred(Aarg_MaSp2.1a most likely was the baseline which evolved into Atri_MaSp2.1a and Aaur_MaSp2.1a as well as Atri_MaSp2.1b most likely was the baseline which evolved into Aarg_MaSp2.1b and Aaur_MaSp2.1b). In addition one can see that there is a lot of relatedness between the spidroin types being that many of the segments of the repeat structure are similar.</p>

                
            <h3>Species Comparison<h3>
                <div class= "box3">
                    <img src="/Aaur_speciesaligncomp.png" alt="auar">
                    <img src="/aarg_speciesaligncomp.png" alt="aarg"></div>
                    <p>In this part of data we are comparing the different repeats units of each species group (Aaur, Atri, Aarg) to each other. There isn't a phylogenetic tree being shown because it would have just been the connection between two of the two sequences  not giving any new information. The alignments however do show the relatedness  between the two spidroin types of the same species. Here we can see that compared to the previous shown data of the spidroin comparison the species data shows less relativity and is less indicative of the data. In addition we can conclude from this that when in correlation of look for which spider spidroin would be more similar it is best to look at the same types of spidrons over the same types of species being there tend to be more similar between the two.</p>

                
            
            <h3>Code Results</h3>
                <div class= "box4"> 
                    <img src="/masp1a_fasta.png" alt="image">
                    <img src="/masp2b_fasta.png" alt="image"></div>
                    <p> Here are the results of the code we wrote which is a fasta file of the largest repeat unit found when going through a fasta file holding the protien sequinece. In this example one can see the same species type largest sequience but with different spidrons types proving more how it is more indicative to look at spidron types for relatively of the protien.</p> 
                
        </div>
    </main>
</body>
</html>


